tap/tracker.py

```diff
@@ -48,10 +53,18 @@ def _set_outdir(self, outdir):
 
     outdir = property(_get_outdir, _set_outdir)
 
+    def _get_header(self):
```

Comment:
Could you drop `_get_header`, `_set_header`, and the `header` property? Right now they aren't doing anything special. If there is a need in the future to have the header be a property and do extra work, then we can reintroduce the property. As is, it's an extra layer of indirection that is not pulling its weight, IMO.